account_banking_payment_transfer/model/account_payment.py

```diff
@@ -197,6 +198,31 @@ def action_sent_no_move_line_hook(self, pay_line):
         """This function is designed to be inherited"""
         return
 
+    @api.model
+    def _create_move_line_partner_account(self, line, move, labels,
+                                          order_type):
+        """This method is designed to be inherited in a custom module"""
+
+        # TODO: take multicurrency into account
+        aml_obj = self.env['account.move.line']
+        # create the payment/debit counterpart move line
+        # on the partner account
+        partner_ml_vals = self._prepare_move_line_partner_account(
+            line, move, labels, order_type)
+        partner_move_line = aml_obj.create(partner_ml_vals)
+
+        # register the payment/debit move line
+        # on the payment line and call reconciliation on it
+        line.write({'transit_move_line_id': partner_move_line.id})
+
+    def _reconcile_payment_lines(self, payment_lines):
+        pl_obj = self.env['payment.line']
+        for line in payment_lines:
+            if line.move_line_id:
+                pl_obj.debit_reconcile(line.id)
```

Comment:
@sbidoul I think debit_reconcile isn't considered like a multi method because there is no ```ids``` parameter. Therefore, ```line.debit_reconcile()``` doesn't work. I can change ```debit_reconcile``` method déclaration to replace ```payment_line_id``` with ```ids``` and integrate a loop in this method